src/muya/lib/index.js

```diff
@@ -334,6 +334,14 @@ class Muya {
     }
   }
 
+  /**
+   * Get all images' src from the given markdown.
+   * @param {string} markdown you want to extract images from this markdown.
+   */
+  extractImages (markdown = this.markdown) {
```

Comment:
Why is the input the markdown text? It should be the block structure or not? Please put this in the `utils` folder if you prefer markdown.

Reply:
It will use the `muya.markdown` in most of our use case, we can also support pass markdown to it, and extract the images in this markdown content which we pass to this function, so we can not put it in the utils, because we use the `this.markdown`?

Reply:
I thought markdown should be completely removed in the new data structure? Muya should only operate on our data structure and be able to export to HTML/markdown but shouldn't generate markdown after each operation like now because it's not needed/waste of resources.

Reply:
It belongs to the refactor in the future.